Pod/Classes/WPMediaCollectionViewCell.m

```diff
@@ -84,13 +90,28 @@ - (void)commonInit
 
     self.selectedBackgroundView = _selectionFrame;
 
-    _captionLabel = [[UILabel alloc] initWithFrame:CGRectMake(0, self.contentView.frame.size.height - counterTextSize, self.contentView.frame.size.width, counterTextSize)];
-    _captionLabel.backgroundColor = [UIColor colorWithWhite:0.2 alpha:0.7];
+    CGFloat labelTextSize = 12.0;
+    CGFloat labelHeight = 30.0;
+    CGFloat labelMargin = 5.0;
```

Comment:
I think according to the designs the label should be 10pt from the right edge and bottom, not 5.